apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/FirestoreMultipleDbTest.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2022, Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.ClientTesting;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests;
+
+public class FirestoreMultipleDbTest
+{
+    private readonly MultipleFirestoreDbFixture _fixture;
```

Comment:
Sorry, I hadn't spotted this before.

This isn't how fixtures are meant to be used. The point of a fixture is for Xunit to instantiate it and pass it to the test. That way the same object can be used for multiple tests etc.

The test class should have the `[Collection(nameof(MultipleFirestoreDbFixture))]` attribute, and it should accept the fixture in a constructor. See RunInTransactionTest for an example.

Reply:
Thanks @jskeet for pointing this out. Will update it.